helm-charts/bytebase/values.yaml

```diff
@@ -9,5 +9,5 @@ bytebase:
     external-url: https://www.bytebase.com/docs/get-started/install/external-url
     pg: postgresql://user:secret@host:port/dbname
     data: /var/opt/bytebase
-  version: 1.3.2
+  version: 1.7.0
```

Comment:
nit: we can think of a way to pull released version later.

Reply:
I want to write an action to bump this version automatically if the version file changed.

Reply:
Also, users can use set command to set this version by themselves likes below:
```bash
$ helm -n bytebase \
--set "bytebase.option.port"=443 \
--set "bytebase.option.external-url"="https://bytebase.com" \
--set "bytebase.option.pg"="postgresql://bytebase:bytebase@database.bytebase.ap-east-1.rds.amazonaws.com/bytebase" \
--set "bytebase.version"=1.7.0 \
install bytebase-release bytebase-repo/bytebase
```